src/pint/models/parameter.py

```diff
@@ -1581,19 +1593,20 @@ def select_toa_mask(self, toas):
         # get the table columns
         # TODO Right now it is only supports mjd, freq, tel, and flagkeys,
         # We need to consider some more complicated situation
+        # THis will have a problem if the key has '-' in it.
```

Comment:
What does this mean about how keys should  be specified? Is `JUMP -tel` allowed as an alternative form of `JUMP tel`? Should it be?